first-steps.mdx

```diff
@@ -1,137 +1,130 @@
 ---
 title: First steps
-description: "Start using octomind immediately, and generate your first test case."
+description: "Generate your first tests and run them. Find bugs in your app before your user does."
 icon: "lightbulb"
 ---
 
-<div
-  style={{
-    position: "relative",
-    paddingBottom: "calc(55.43121693121693% + 42px)",
-    height: 0,
-  }}
->
-  <iframe
-    src="https://app.supademo.com/embed/rSpeR5F3cFGeg0EHBPQ2E"
-    allow="clipboard-write"
-    frameborder="0"
-    webkitallowfullscreen="true"
-    mozallowfullscreen="true"
-    allowfullscreen
-    style={{
-      width: "100%",
-      height: "100%",
-      borderRadius: 12,
-      position: "absolute",
-      top: 0,
-      left: 0,
-    }}
-  ></iframe>
-  <div
-    style={{
-      height: 16,
-      backgroundColor: "#13142c",
-      inset: "auto 0px 0px 0px",
-      position: "absolute",
-    }}
-  />
-</div>
-
 ## 1. Give us a URL
 
-We'll ask for a URL to create test cases. The URL has to be publicly accessible.
+We'll ask for a URL to create test cases. The URL has to be publicly accessible. We can test both in staging and in production, as long as we can access the site.
 
-<Frame caption="First page of the setup flow - link to your website, screenshot 09/2023">
+<Frame caption="First page of the setup flow - link to your website, screenshot 07/2024">
   <img src="/images/setup/setup-1-url.png" alt="enter your url screen" />
 </Frame>
 
 ## 2. Name your project
 
-The second page will render a screenshot of the url you provided. It tests that your URL is accessible.
-Now, you need to give your project a name. We'll propose one similar to the url you provided.
+The second page will render a screenshot of the url you provided. We chexk if it is accessible. You can name your project now. We'll propose one similar to the url you provided.
 But you can choose your own.
 
-<Frame caption="Second page of the setup flow: Name your project, screenshot 09/2023">
+<Frame caption="Second page of the setup flow: Name your project, screenshot 07/2024">
   <img
     src="/images/setup/setup-2-project.png"
-    alt="name your project screen, screenshot 09/2023"
+    alt="name your project screen, screenshot 07/2024"
   />
 </Frame>
 
 <div class="mt-8" />
 
-## 3. Select test cases to be generated
+## 3. Sign up
```

Comment:
Decided to not mention "confirm your email/ set password" flow?

Reply:
It's kinda implied, I didn't wanna overstretch the steps.